### PR TITLE
Fix LSTM and GRU parameters description

### PR DESCRIPTION
**Pull request by cdiep**

### CODE REVIEW

**Review by ezyang — APPROVED** [review]

File: torch/nn/modules/rnn.py [hunk 1]
```diff
@@ -341,10 +341,11 @@ class LSTM(RNNBase):
             \end{array}
 
     where :math:`h_t` is the hidden state at time `t`, :math:`c_t` is the cell
-    state at time `t`, :math:`x_t` is the hidden state of the previous layer at
-    time `t` or :math:`input_t` for the first layer, and :math:`i_t`,
-    :math:`f_t`, :math:`g_t`, :math:`o_t` are the input, forget, cell,
-    and out gates, respectively. :math:`\sigma` is the sigmoid function.
+    state at time `t`, :math:`x_t` is the input at time `t`, :math:`h_{(t-1)}`
+    is the hidden state of the previous layer at time `t` or the initial hidden
```
Comment (SsnL) [inline torch/nn/modules/rnn.py]: nit: should be  ``at time `t-1` ``

File: torch/nn/modules/rnn.py [hunk 2]
```diff
@@ -425,10 +426,11 @@ class GRU(RNNBase):
             h_t = (1 - z_t) n_t + z_t h_{(t-1)} \\
             \end{array}
 
-    where :math:`h_t` is the hidden state at time `t`, :math:`x_t` is the hidden
-    state of the previous layer at time `t` or :math:`input_t` for the first
-    layer, and :math:`r_t`, :math:`z_t`, :math:`n_t` are the reset, input,
-    and new gates, respectively. :math:`\sigma` is the sigmoid function.
+    where :math:`h_t` is the hidden state at time `t`, :math:`x_t` is the input
+    at time `t`, :math:`h_{(t-1)}` is the hidden state of the previous layer
+    at time `t` or the initial hidden tate for the first layer, and :math:`r_t`,
```
Comment (SsnL) [inline torch/nn/modules/rnn.py]: same here

**Review by ssnl — CHANGES_REQUESTED** [review]
Please fix the nits

**Comment by SsnL** [comment]
Thank for your PR.

I'm going to stop the CI run for this as they don't test for docs anyways. Please address the two minor comments. :)

File: torch/nn/modules/rnn.py [hunk 3]
```diff
@@ -425,10 +426,11 @@ class GRU(RNNBase):
             h_t = (1 - z_t) n_t + z_t h_{(t-1)} \\
             \end{array}
 
-    where :math:`h_t` is the hidden state at time `t`, :math:`x_t` is the hidden
-    state of the previous layer at time `t` or :math:`input_t` for the first
-    layer, and :math:`r_t`, :math:`z_t`, :math:`n_t` are the reset, input,
-    and new gates, respectively. :math:`\sigma` is the sigmoid function.
+    where :math:`h_t` is the hidden state at time `t`, :math:`x_t` is the input
+    at time `t`, :math:`h_{(t-1)}` is the hidden state of the previous layer
+    at time `t-1` or the initial hidden tate for the first layer, and :math:`r_t`,
```
Comment (SsnL) [inline torch/nn/modules/rnn.py]: Sorry, but could you also change `the first layer` to ``at time `0` ``? Same for LSTM.

Comment (cdiep) [reply torch/nn/modules/rnn.py]: Make sense. Done.  Thanks for your review!

**Review by ssnl — APPROVED** [review]
Thanks!

**Pull request merged**